rules.md

```diff
@@ -0,0 +1,18 @@
+# Substrate
+
+- 🔒️ed lines are reviewed by 2 of {@gavofyork, @shawntabrizi, @rphmeier, @bkchr}
```

Comment:
```suggestion
- 🔒️ed lines are reviewed by 2 team leads, one of them must be in the group {@gavofyork, @shawntabrizi, @rphmeier, @bkchr}
```

Reply:
These changes including pinging the people?

Reply:
yes, as we've discussed
```suggestion
- 🔒️ed lines are reviewed by 2 team leads, one of them must be in the group {@gavofyork, @shawntabrizi, @rphmeier, @bkchr} and the other in another team, @paritytech/polkadotleads 
```